### ISSUE TITLE
Zooniverse vote

### ISSUE THREAD

**Issue by iaugs** [stuartlynn/nameexoworlds.org#2]
## Pages

```
Welcome back zooniverse

You can now start voting on systems to be named.
```

Comment: I don’t understand "Welcome back zooniverse" ?

After "You can now start voting on systems to be named" we have to mention a deadline for closing the vote. In the Process, the next step starts in February.

So we may write:

```
You can now start voting on systems to be named

The vote will close on 31 January, 23h59m59s UT
```

This can be repeated:

zooniverse

We are asking you to vote on systems to be taken to the next stage of the naming proces. You can vote for as many systems as you like but only once on each system

The vote will close on 31 January, 23h59m59s UT
- The "see your votes" page lists them in the order of voting --> re-order in the same order as in the list ?
## Corrections
- As an orginisation we need your help to select systems to open to a public vote. You can vote for as many of the systems as you like but only once per system
  
  orginisation --> organisation

(and the "As" is cut from the sentence but appears in isolation at the top right corner of the page)
- Add reference:

[1] Allen, R.H., 1899, "Star Names: their lore and their meaning" (Dover Publications, rev. ed. 1963); Kunitzsch, P., and Smart, T., 2006, "A Dictionary of Modern Star Names" (Shy Publishing, Cambridge, MA, USA)
- Star details
  Constallation: the Hercules
  Vmag: 6.7
  Visibility: Visible through binocular
  No planets: 1
  Namable: Yes
  
  Constallation --> Constellation
  
  the Hercules --> Hercules
  
  No planets --> Number of planets ["No" is misleading and could be taken as "no"]
- Planet details
  55 Cancri b
  Mass (jupiters): 0.8
  Mass (earths): 254.3
  Period (days): 14.65
  Semi Major Axis (AU): 0.1134
  
  jupiters --> Jupiters [capital "J"]
  earths   --> Earths [capital "E"]
- Remove vote for this system --> Remove your vote for this system
- Corrections to constellations in the list:
  
  the Hercules --> Hercules
  
  the Bernice’s hair --> Berenice’s hair
